CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- Added 'skipResetOffset' to disable/enable bypassing of resetOffset rpc call for setting FT sensor offsets to zero. (See [!72](https://github.com/robotology/whole-body-estimators/pull/72)).
```

Comment:
```suggestion
- Added 'startWithZeroFTSensorOffsets' to disable/enable bypassing of resetOffset rpc call for setting FT sensor offsets to zero. (See [!72](https://github.com/robotology/whole-body-estimators/pull/72)).
```

Reply:
Addressed in https://github.com/robotology/whole-body-estimators/pull/72/commits/52813c1dfd0470723a4703d4ff165b81e76bcea6